classifiers/dns.go

```diff
@@ -0,0 +1,30 @@
+package classifiers
+
+import (
+	"github.com/google/gopacket/layers"
+	"github.com/mushorg/go-dpi"
+)
+
+type DnsClassifier struct{}
+
+func (_ DnsClassifier) HeuristicClassify(flow *godpi.Flow) bool {
+	if len(flow.Packets) == 0 {
+		return false
+	}
+	for _, packet := range flow.Packets {
+		if layer := (*packet).Layer(layers.LayerTypeUDP); layer != nil {
+			srcPort := layer.(*layers.UDP).SrcPort
+			dstPort := layer.(*layers.UDP).DstPort
+			if srcPort != 53 && dstPort != 53 {
```

Comment:
Sorry, my bad, I had the logic the wrong way around where you detect DNS instead of detecting if it is not DNS :+1: